specification/synapse/resource-manager/Microsoft.Synapse/stable/2020-12-01/sqlServer.json

```diff
@@ -1089,13 +1089,13 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/recoverableSqlpools": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/recoverableSqlPools": {
```

Comment:
Can you let me know why change this case issue? as it will cause SDK breaking change, specially for operation id change. if not special reason. suggest to revert.

Reply:
The SDK's have not yet been generated for this api version. This is a new API version we're adding and there are NO SDKs right now. So nothing would fail.

Reply:
@akning-ms Is there anything pending for merging this PR?